CONTRIBUTING.md

```diff
@@ -48,6 +48,10 @@ be accepted. When in doubt, make sure to first raise an
 
 To the extent possible, the pull request process guards our coding guidelines.
 Some pointers:
+- Make sure that the `mvn clean install` build fully passes, ideally before
+  opening a pull request. See the [development
```

Comment:
Ideally? Or should we make this more mandatory

```suggestion
- Before opening a Pull Request, make sure that the `mvn clean install` 
build fully passes. See the [development
```

Reply:
I was actually thinking about this last night, and realized that if people get stuck, this requirement might discourage them from contributing at all. I proposed a new sentence; PTAL :)